src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/PreviewTokenControllerTests.cs

```diff
@@ -3,34 +3,72 @@
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
 using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
-using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
-using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
-using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.AspNetCore.WebUtilities;
 using Microsoft.EntityFrameworkCore;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Controllers.Api.Public.Data;
 
-public abstract class PreviewTokenControllerTests(TestApplicationFactory testApp) : IntegrationTestFixture(testApp)
+// TODO EES-6450 - decide if we're happy with ignoring this #pragma in test projects.
+// ReSharper disable once ClassNeverInstantiated.Global
+#pragma warning disable CS9107 // Parameter is captured into the state of the enclosing type and its value is also passed to the base constructor. The value might be captured by the base class as well.
+public class PreviewTokenControllerTestsFixture()
+    : OptimisedAdminCollectionFixture(
+        capabilities: [AdminIntegrationTestCapability.UserAuth, AdminIntegrationTestCapability.Postgres]
+    );
+
+[CollectionDefinition(nameof(PreviewTokenControllerTestsFixture))]
+public class PreviewTokenControllerTestsCollection : ICollectionFixture<PreviewTokenControllerTestsFixture>;
+
+[Collection(nameof(PreviewTokenControllerTestsFixture))]
+public abstract class PreviewTokenControllerTests(PreviewTokenControllerTestsFixture fixture) : IAsyncLifetime
 {
     private const string BaseUrl = "api/public-data/preview-tokens";
+    private static readonly DataFixture DataFixture = new();
+
+    private static bool _commonDataInitialized;
+
+    private static readonly ClaimsPrincipal BauUser = DataFixture.BauUser();
 
-    private static readonly ClaimsPrincipal BauUser = new DataFixture().BauUser();
+    private static readonly User BauUserEntry = DataFixture
+        .DefaultUser()
+        .WithId(BauUser.GetUserId())
+        .WithEmail(BauUser.GetEmail());
+
+    /// <summary>
+    /// Set up common test data that is commonly used by all tests within this suite.
+    /// TODO EES-6450 - see if we're happy with this way of setting up global test suite test data.
+    /// </summary>
+    public async Task InitializeAsync()
+    {
+        if (!_commonDataInitialized)
+        {
+            fixture.RegisterTestUser(BauUser);
+            await fixture.GetContentDbContext().AddTestData(context => context.Users.Add(BauUserEntry));
+            _commonDataInitialized = true;
+        }
+    }
```

Comment:
Just wondering if there are any alternatives to this as it means `PreviewTokenControllerTests` is quite a bit different to the other tests in the way that the base class needs the `fixture` parameter which has an affect on all the sub-classes and causes the 'captured state' warning?